api-connection.html

```diff
@@ -138,8 +138,10 @@ <h2 id="function_newConnection">new Connection(config)</h2>
           <ul>
             <li>username (string): A user needs to provide the `userName` associated to their account.</li>
             <li>password (string): A user needs to provide the `password` associated to their account.</li>
-            <li>clientId (string): Application (`client`) ID from your registered Azure application. This currently is optional
-              and uses a default clientId, but a user-provided clientId will be required in the near future.
+            <li>clientId (string): Application (`client`) ID from your registered Azure application. 
+              The application has to have public client flows enabled and has to have `user_impersonation` 
+              permissions for Azure SQL Database. The user account that is being used also has to be assigned
+              to the application, and has to grant consent (either explicitly or via admin consent).
             </li>
```

Comment:
I also remember David mentioned that our pervious hard coded clientID will be expired, so if there are users saved and keep using that client id, it won't work in the future. They have to grad their own clientID and use that for connections. We should also mention this as well.